src/aws-cpp-sdk-core/source/config/EC2InstanceProfileConfigLoader.cpp

```diff
@@ -38,6 +39,14 @@ namespace Aws
             }
         }
 
+        EC2InstanceProfileConfigLoader::EC2InstanceProfileConfigLoader(const Aws::Client::ClientConfiguration& clientConfig)
```

Comment:
i dont this we want to add a constructor for `Aws::Client::ClientConfiguration` we only want to add one for `CredentialProviderConfiguration`